linkserver/src/main.rs

```diff
@@ -0,0 +1,162 @@
+#[macro_use]
+extern crate actix;
+extern crate actix_web;
+extern crate futures;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+extern crate rand;
+extern crate uuid;
+
+use actix::*;
```

Comment:
not a fan of the star import. I see actix even has a prelude to encourage it (that's used in server.rs), not a fan of that either